src/app/plan/components/Wizard/HooksFormComponent.tsx

```diff
@@ -0,0 +1,442 @@
+                    >
+                      {/*
+          // @ts-ignore issue: https://github.com/konveyor/mig-ui/issues/747 */}
+                      <TextInput
+                        onChange={formikHandleChange}
+                        onInput={formikSetFieldTouched(destServiceAccountNameKey)}
+                        onBlur={handleBlur}
+                        value={values.destServiceAccountName}
+                        name={destServiceAccountNameKey}
+                        type="text"
+                        id="dest-service-account-name-input"
+                        isValid={!(touched.destServiceAccountName && errors.destServiceAccountName)}
+                      />
+                    </FormGroup>
+                  </GridItem>
+                  <GridItem className={spacing.mtSm}>
+                    <FormGroup
+                      label="Service account namespace"
+                      isRequired
+                      fieldId="destServiceAccountNamespace"
+                      helperTextInvalid={touched.destServiceAccountNamespace && errors.destServiceAccountNamespace}
+                      isValid={!(touched.destServiceAccountNamespace && errors.destServiceAccountNamespace)}
+                    >
+                      {/*
+          // @ts-ignore issue: https://github.com/konveyor/mig-ui/issues/747 */}
+                      <TextInput
+                        onChange={formikHandleChange}
+                        onInput={formikSetFieldTouched(destServiceAccountNamespaceKey)}
+                        onBlur={handleBlur}
+                        value={values.destServiceAccountNamespace}
+                        name={destServiceAccountNamespaceKey}
+                        type="text"
+                        id="dest-service-account-namespace-input"
+                        isValid={!(touched.destServiceAccountNamespace && errors.destServiceAccountNamespace)}
+                      />
+                    </FormGroup>
+                  </GridItem>
+                </React.Fragment>
+              }
+            </Grid>
+          </FormGroup>
+        </GridItem>
+        <GridItem className={spacing.mtMd}>
+          <FormGroup
+            label="Migration step when the hook should be run"
+            isRequired
+            fieldId={migrationStepKey}
+            className={styles.styledFormGroupTitle}
+            helperTextInvalid={touched.migrationStep && errors.migrationStep}
+            isValid={!(touched.migrationStep && errors.migrationStep)}
+          >
+            <SimpleSelect
+              id="migrationStep"
+              onChange={(value) => {
+                setFieldValue('migrationStep', value);
+                setFieldTouched('migrationStep');
+              }}
+              options={phaseOptions}
+              value={values.migrationStep}
+              placeholderText="Select phase..."
+            />
+          </FormGroup>
+        </GridItem>
+        <Grid span={6}>
```

Comment:
We can leave it, but I wonder if we're overusing `Grid` in general. Two buttons directly in a div with some spacer classes on them would probably do fine here.

Reply:
+1 lets open a seperate issue to fix this app wide. Agree that we are using it too much & probably incorrectly.